packages/server/test/unit/config_spec.js

```diff
@@ -690,6 +690,33 @@ describe('lib/config', () => {
         })
       })
 
+      context('retries', () => {
+        const retriesError = 'a positive number or null or an object with keys "openMode" and "runMode" with values of numbers or nulls'
+
+        // need to keep the const here or it'll get stripped by the build
+        // eslint-disable-next-line no-unused-vars
+        const cases = [
+          [{ retries: 3 }, 'when a number', true],
+          [{ retries: true }, 'when true', false],
+          [{ retries: false }, 'when false', false],
+          [{ retries: -1 }, 'with a negative number', false],
+          [{ retries: {} }, 'with an empty object', true],
+          [{ retries: null }, 'with null', true],
+          [{ retries: { runMode: 3 } }, 'when runMode is a positive number', true],
+          [{ retries: { runMode: -1 } }, 'when runMode is a negative number', false],
+          [{ retries: { openMode: 3 } }, 'when openMode is a positive number', true],
+          [{ retries: { openMode: -1 } }, 'when openMode is a negative number', false],
+          [{ retries: { openMode: 3, TypoRunMode: 3 } }, 'when there is an additional unknown key', false],
+          [{ retries: { openMode: 3, runMode: 3 } }, 'when both runMode and openMode are positive numbers', true],
```

Comment:
This kind of table-style `test` statement is common in jest for dealing with validating many cases... I hope this is ok 😅

Reply:
I think it's good.